### ISSUE TITLE
Qwen2模型的vllm部署--tensor-parallel-size问题

### ISSUE THREAD

**Issue by xuexidi** [QwenLM/Qwen2#668]
您好，非常感谢您的项目组开源了如此强大的大模型！Qwen、Qwen1.5系列我们一直在沿用，受益匪浅！

问题：
当我想要用vllm部署近期开源的Qwen2-57B-A14B-chat时，我的启动指令如下：

**Comment by jklj077** [comment]
I completely agree with you on that it is quite unfortunate that Qwen2-57B-A14B cannot be used with tp=8, which is also the case with Qwen2-7B. The number of attention heads is determined by a series of experiments aiming for better performance, but it does lead to an unintentional side-effect. However, I must say that it would be very difficult and inefficient to make it work on GPUs with less than 40GB memory.

For our plan on the model scales, please also refer to the comment at <https://github.com/QwenLM/Qwen2/issues/482#issuecomment-2172125101>.
